packages/store/tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
     "extends": "../../tsconfig.json",
-    "include": ["src", "test"],
+    "include": ["src"],
     "compilerOptions": {
         "baseUrl": "./src",
+        "rootDir": "src",
         "outDir": "dist",
-        "strict": false,
         "paths": {
             "types/*": ["types/*"],
             "utils/*": ["utils/*"]
         }
```

Comment:
```suggestion

```
Related to this: https://github.com/SAP/open-ux-tools/pull/798/files#r1022164523

Reply:
removed